packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/cupertino.dart';
+
+void main() {
+  const double _kOpenScale = 1.2;
+  final UniqueKey childKey = UniqueKey();
+  final Widget child = Container(
+    key: childKey,
+    width: 100.0,
+    height: 100.0,
+    color: CupertinoColors.activeOrange,
+  );
+
+  Widget _getContextMenu() {
+    return CupertinoApp(
+      home: CupertinoPageScaffold(
+        child: Center(
+          child: ContextMenu(
+            actions: const <ContextMenuSheetAction>[
+              ContextMenuSheetAction(
+                child: Text('ContextMenuSheetAction'),
+              ),
+            ],
+            child: child,
+          ),
+        ),
+      ),
+    );
+  }
+
+  // Finds the child widget that is rendered inside of _DecoyChild.
+  Finder _findDecoyChild(Widget child) {
+    return find.descendant(
+      of: find.byType(ShaderMask),
+      matching: find.byWidgetPredicate((Widget w) => w == child),
```

Comment:
I think this is equivalent to `find.byWidget(child)`?

Reply:
You appear to be right, thanks for the tip.